File: LV1/1.3/batterup/batterup.go
package main

import "fmt"

func main() {
	var n int
	var sum, l, temp float64

	fmt.Scanf("%d", &n)

	for i := 0; i < n; i++ {
		fmt.Scanf("%f", &temp)
		if temp >= 0 {
			sum += temp
			l++
		}
	}
	fmt.Println(sum / l)
}

File: LV1/1.3/batterup/batterup.cpp
#include <cstdio>

int main() {
    int n;
    double sum, l, temp;

    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        scanf("%lf", &temp);
        if (temp >= 0) {
            sum += temp;
            l++;
        }
    }
    printf("%lf\n", sum / l);
}
